media/css/m24/components/navigation-refresh.scss

```diff
@@ -30,6 +30,7 @@ $margin-top: 54px; // top margin offset for mobile navigation menu
     background-color: $color-white;
     border-bottom: $border-width solid $m24-color-light-mid-gray;
     display: flex;
+    line-height: var(--body-line-height);
```

Comment:
Hmh now with #15803 also affecting footer, and the letterspacing mentioned there too, I'm starting to think whether instead of adding more and more explicit `line-height` and `letter-spacing` values around here for specificity a better approach would be to only apply all of the generic m24-base styles specifically to something like `#outer-wrapper > main` scope, not to interfere with header and footer…?